docs/understanding_deis/components.rst

```diff
@@ -27,7 +27,9 @@ platform state. Backups and WAL logs are pushed to :ref:`Store`.
 
 Cache
 -----
-The cache is an instance of `Redis`_ used by the registry.
+The cache component is an optional instance of `Redis`_. If the cache is running,
+it will be used by the :ref:`registry`. The ``deisctl`` provisioning tool does
```

Comment:
Yeah, I'd add a brief sentence about why we disabled it, and why one might want to enable it.